tools/scan-build-py/README.md

```diff
@@ -41,6 +41,32 @@ goes like this:
 Use `--help` to know more about the commands.
 
 
+How to use the experimental Cross Translation Unit analysis
+-----------------------------------------------------------
+
+To run the CTU analysis, a compilation database file has to be created:
+
+    $ intercept-build <your build command>
+
+To run the Clang static analyzer against a compilation database
+with CTU analysis enabled, execute:
+    
+    $ analyze-build --ctu
+
+For CTU analysis an additional (function-definition) collection-phase is needed. 
+For debugging purposes, it is possible to separately collect and analyze 
+the output of the Clang static analyzer. By doing this, the intermediate files
```

Comment:
1. Nit: Clang Static Analyzer
2. ".. collect and analyze the output of .." Could you explain this to me since this does not make sense to me right now. We collect stuff and then use it for the analysis, however, the output of of the CSA is not used by this feature if I understand it right.